infer/src/base/Config.ml

```diff
@@ -2725,6 +2730,8 @@ let process_linters_doc_url args =
   fun ~linter_id -> List.Assoc.find ~equal:String.equal linter_doc_url_assocs linter_id
 
 
+let is_child = !run_as_child >= 0
+
```

Comment:
Please move at the bottom of the file after the options have been "frozen", together with other values that depend on option values. Ah, but it's needed before... See my other comment, not sure why that's needed.

Reply:
I introduced the boolean to make the code outside of `Config` better-looking, as `Config.run_as_child >= 0` looked very weird and unclear. With the `int option` option, this should be better. Will update.

Reply:
Actually this entirely disappears.

Reply:
Can be resolved.